learnyounode/05-module.js

```diff
@@ -0,0 +1,20 @@
+var fs = require('fs');
+
+filterList = function (err, list) {
+  list.forEach(function(cV, i, array) {
+    if (cV.endsWith('.' + process.argv[3])) {
+      console.log(cV);
+    }
+  })
+}
+
+// var path = process.argv[2];
+// var fileExt = process.argv[3];
+
+filterListLog = function() {
+  fs.readdir(process.argv[2], filterList)
+}
+
+module.exports = {
+  filterListLog
+}
```

Comment:
This is not the right idea, and I'm not sure it will even work; `process.argv` may not be available to the interior module.

The point of having a module is for the code to be broken apart. Since `05-make-it-modular.js` is run from the command line, that's where `process.argv` should be handled. The module's job should just be to do the filtering. Even printing can and probably should happen in `make-it-modular.js`, since that would allow the module to be used in a wider variety of contexts.